.github/workflows/release.yml

```diff
@@ -79,6 +79,27 @@ jobs:
           password: ${{ secrets.PYPI }}
     needs:
       - test
+
+  build-docs:
```

Comment:
Shouldn't this not depend on releasing a new package?  What if there is an error in the docs we would have to make a whole new ipfabric version?

Reply:
agreed, moved to its own yml file see commit https://github.com/community-fabric/python-ipfabric/pull/214/commits/fe3764421b0183bb50d2e38009bc4e6ef2fe1b69